routes/api.js

```diff
@@ -7,6 +7,14 @@ var debug = require('debug')('dashboard-proxy:server');
 var error = require('debug')('dashboard-proxy:server:error');
 var Buffer = require('buffer').Buffer;
 var BufferList = require('../node_modules/websocket/vendor/FastBufferList');
+var BufferList;
+try {
+    // from source tree
+    BufferList = require('../node_modules/websocket/vendor/FastBufferList');
+} catch(e) {
+    // from install location
+    BufferList = require('../../websocket/vendor/FastBufferList');
+}
```

Comment:
The `install-test` target works fine for me, but when I installed manually into a Docker container and ran, I get the following error:

```
[root@fa640848f741:/tmp# ./node_modules/.bin/jupyter-dashboards-server                          
module.js:341
    throw err;
    ^

Error: Cannot find module '../node_modules/websocket/vendor/FastBufferList'
    at Function.Module._resolveFilename (module.js:339:15)
    at Function.Module._load (module.js:290:25)
    at Module.require (module.js:367:17)
    at require (internal/module.js:16:19)
    at Object.<anonymous> (/tmp/node_modules/jupyter-dashboards-server/routes/api.js:9:18)
    at Module._compile (module.js:413:34)
    at Object.Module._extensions..js (module.js:422:10)
    at Module.load (module.js:357:32)
    at Function.Module._load (module.js:314:12)
    at Module.require (module.js:367:17)
[root@fa640848f741:/tmp# ./node_modules/jupyter-dashboards-server/bin/jupyter-dashboards-server 
module.js:341
    throw err;
    ^

Error: Cannot find module '../node_modules/websocket/vendor/FastBufferList'
    at Function.Module._resolveFilename (module.js:339:15)
    at Function.Module._load (module.js:290:25)
    at Module.require (module.js:367:17)
    at require (internal/module.js:16:19)
    at Object.<anonymous> (/tmp/node_modules/jupyter-dashboards-server/routes/api.js:9:18)
    at Module._compile (module.js:413:34)
    at Object.Module._extensions..js (module.js:422:10)
    at Module.load (module.js:357:32)
    at Function.Module._load (module.js:314:12)
    at Module.require (module.js:367:17)
```